src/kaleidoscope/key_defs.h

```diff
@@ -255,14 +260,19 @@ typedef kaleidoscope::Key Key_;
 #define KEY_LEFT_FN2 0xff
 #define Key_LFN2 Key(KEY_LEFT_FN2, KEY_FLAGS)
 
+#define SYSTEM_KEY(code, hid_type) \
+  Key(code, SYNTHETIC | IS_SYSCTL | (hid_type & HID_TYPE_MASK))
 
 /* Most Consumer keys are more then 8bit, the highest Consumer hid code
    uses 10bit. By using the 11bit as flag to indicate a consumer keys was activate we can
    use the 10 lsb as the HID Consumer code. If you need to get the keycode of a Consumer key
    use the CONSUMER(key) macro this will return the 10bit keycode.
 */
-#define CONSUMER(key) (key.getRaw() & 0x03FF)
-#define CONSUMER_KEY(code, flags) Key((code) | ((flags | SYNTHETIC|IS_CONSUMER) << 8))
+constexpr uint16_t CONSUMER_KEYCODE_MASK = 0x03FF;
```

Comment:
Magic number replaced with defined constant. The official code style guide says it should be named `kConsumerKeycodeMask`, but that style isn't used anyplace else in Kaleidoscope that I can find, so I'm leaving it in all caps for the moment.